File: aula3/ex1.cpp
#include <iostream>

using namespace std;

int main () {
    int num;
    cout << "Num: " << endl;
    cin >> num;
    if(num % 2 == 0) {
        cout << "Par" << endl;
    }
    cin >> num;

    return 0;
}

File: aula5/ex1/Ex1.java
import java.util.Scanner;

class Ex1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int num;
        System.out.println("Digite o seu número: ");
        num = scanner.nextInt();
        if(num % 2 == 0) {
            System.out.println("Número par");
        } else {
            System.out.println("Número impar");
        }
        num = scanner.nextInt();
    }
}